lighthouse-core/report/templates/report-template.html

```diff
@@ -46,6 +46,13 @@
             <a href="#" data-action="save-json">Save as JSON...</a>
           </ul>
         </span>
+        <span class="rerun-section">
+          <button class="report-body__icon rerun-button js-rerun-button"></button>
+          <div class="rerun-popup js-rerun-popup" status="idle">
+            <p class="rerun-message">Rerunning Lighthouse</p>
+            <a class="rerun-report-link js-rerun-report-link" href="/?id=rerun">New Report</a>
```

Comment:
Thanks for the advice. The button now rotates while rerunning. Also tried graying out the background, but that does not look very nice.